src/riscv-integration.adoc

```diff
@@ -424,6 +424,23 @@ include::generated/csr_renamed_purecap_mode_vs_table_body.adoc[]
 include::generated/csr_renamed_purecap_mode_u_table_body.adoc[]
 |===
 
+=== CHERI fault reporting
+
+CHERI faults require additional information to be reported.
+The CSRs updated depend on the mode the trap is taken into, as shown in
+<<cheri-fault-reporting>>.
+
+The additional information is written for CHERI faults and <<cheri_pte_ext,CHERI PTE page faults>>.
```

Comment:
```suggestion
The additional information is written for CHERI faults and <<cheri_pte_ext,CHERI PTE page faults>>, and 
is otherwise written to zero for all other exceptions, except as listed otherwise by other
future extensions.
```